runtime/Go/antlr/token.go

```diff
@@ -208,7 +208,7 @@ func (c *CommonToken) String() string {
 		ch = ""
 	}
 
-	return "[" + "channel:" + strconv.Itoa(c.channel) + " @" + strconv.Itoa(c.tokenIndex) + "," + strconv.Itoa(c.start) + ":" + strconv.Itoa(c.stop) + "='" +
```

Comment:
Hey @willfaught. I changed this local and pushed it without realising that the format needs to be consistent across all targets. Therefore reverted it.